external-crates/move/crates/move-cli/src/base/test.rs

```diff
@@ -67,6 +67,10 @@ pub struct Test {
     /// The number of iterations to run each test that uses generated values (only used with #[random_test]).
     #[clap(name = "rand-num-iters", long = "rand-num-iters")]
     pub rand_num_iters: Option<u64>,
+
+    // Enable tracing for tests
+    #[clap(long = "trace-execution", value_name = "PATH")]
+    pub trace_execution: Option<Option<String>>,
```

Comment:
Assuming you get `Some(None)` from passing just `--trace-execution`, and it means that you want to trace, but you want the tool to pick a filename for you?

Could you make the comment on this flag a doc comment so it gets picked up in the help and add an explanation about this detail?